frontend/src/components/utils/tags/TagList.tsx

```diff
@@ -0,0 +1,53 @@
+import { api, PaginatedResponse, useBetterPaginatedQuery, useBetterQuery } from "../../../services/apiService";
+import React from "react";
+import { Tag as TagComponent } from "./Tag";
+import { Tag } from "../../../models/tag";
+
+export enum Models {
+    Association = "association",
+    Loanable = "loanable",
+    Media = "media",
+    Page = "page",
+    Product = "product",
+    Role = "role",
+}
+
+/**
+ * @param model a `Models` value
+ * @param id the id of the model
+ * @param collapsed true if we want the tags to take less space
+ * @constructor
+ */
+export const TagList = ({ model, id, collapsed = false }) => {
+    let params: any = {};
+    params[model] = id;
+    const { data: tags, status, error } = useBetterQuery<PaginatedResponse<Tag[]>>(
+        ["tags.list", params],
+        api.tags.list,
+    );
+
+    if (status === "loading")
+        return <p className={"text-center"}>Chargement des tags...</p>;
```

Comment:
Could you use the Loading component (and maybe refactor it a little so it can display a text)?